tutorials/full_ecModel/code/sumProtein.m

```diff
@@ -5,9 +5,9 @@
 % Benjamin Sanchez. Last update: 2018-10-27
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
The documentation of this function should be updated. The authorship info is no longer relevant.